rate-limiting/rate-limiting.go

```diff
@@ -0,0 +1,51 @@
+package main
```

Comment:
filenames and package names should use underscores `file_name.go` and not dashes `file-name.go`